pkg/nuctl/test/function_test.go

```diff
@@ -1182,6 +1183,83 @@ func (suite *functionDeployTestSuite) TestDeployWithSecurityContext() {
 		fsGroup))
 }
 
+func (suite *functionDeployTestSuite) TestDeployServiceTypeClusterIPWithInvocation() {
+
+	// TODO: remove this if we ever implement "ServiceType" for local platform
+	suite.ensureRunningOnPlatform("kube")
+
+	uniqueSuffix := "-" + xid.New().String()
+	functionName := "deploy-reverser" + uniqueSuffix
+	imageName := "nuclio/processor-" + functionName
+	functionClusterURL := fmt.Sprintf("http://nuclio-%s.%s.svc.cluster.local:8080",
```

Comment:
You can use `GetDomainNameInvokeURL` to compile you such url
and `ServiceNameFromFunctionName` to generate the service name